docs/UserGuide.md

```diff
@@ -69,18 +69,15 @@ Format: `help`
 
 ### Adding an expense : `add`
 
-Adds an expense to the list.
+Adds an expense to the list. Description and expense category provided must be in alphanumerical format, amount can only be positive number up to 2 decimal places
+and expense date must be in yyyy-MM-dd format.
 
-Format: `add d/EXPENSE_TITLE a/AMOUNT c/DESCRIPTION`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-An expense can have any number of tags (including 0)
-</div>
+Format: `add d/DESCRIPTION a/AMOUNT c/EXPENSECATEGORy t/EXPENSEDATE`
```

Comment:
y for the EXPENSECATEGORY should be capital to to maintain uniformity, and may want to add an underscore between two words, eg EXPENSE_CATEGORY